tests/models/decoder_only/vision_language/vlm_utils/model_utils.py

```diff
@@ -497,6 +497,17 @@ def _generate(self, *args, **kwargs):
     return hf_model
 
 
+def minicpmv_patch_hf_runner(hf_model: HfRunner) -> HfRunner:
```

Comment:
```suggestion
def minicpmo_patch_hf_runner(hf_model: HfRunner) -> HfRunner:
```

Should also update the page containing the tests.